tests/integration/examples/pytorch.py

```diff
@@ -0,0 +1,29 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import pytest
+
+from tests.integration.examples.utils import run_example
+
+
+def test_example(request: pytest.FixtureRequest) -> None:
+    """Runs the pytorch example."""
+    with run_example(
+        request=request,
+        name="pytorch",
+        pipeline_name="fashion_mnist_pipeline",
+        step_count=3,
+        run_count=1,
+    ):
+        pass
```

Comment:
I would suggest to remove any tests that are not implemented as they might give a false sense of security if e.g. PyTorch integration test passes in the CI even though in reality it was not tested

Reply:
Ah, now I get it, the context `with run_example(...)` actually runs the example here and `pass` just means that we don't want to do any post-execution checks afterwards 👍